packages/api/src/constants.js

```diff
@@ -0,0 +1,10 @@
+export const JWT_ISSUER = 'nft-storage'
+export const USER_TAGS = {
+  ACCOUNT_RESTRICTION: 'HasAccountRestriction',
+  SUPER_HOT_ACCESS: 'HasSuperHotAccess',
+}
+export const FAR_FUTURE = new Date('3000-01-01T00:00:00.000Z').getTime()
+export const PAD_LEN = FAR_FUTURE.toString().length
+
+// KV maximum length is 512 bytes - we need to store space for user ID and timestamp
```

Comment:
For reference a non path URL has 'https://bafkreidyeivj7adnnac6ljvzj2e3rd5xdw3revw4da7mx2ckrstapoupoq.ipfs.nftstorage.link/' has 89 bytes, so this should be enough for 99% of the cases.